src_assets/common/assets/web/config.html

```diff
@@ -765,11 +765,18 @@ <h1 class="my-4">Configuration</h1>
       </div>
     </div>
   </div>
-  <div class="alert alert-success my-4" v-if="success">
-    <b>Success!</b> Restart Sunshine to apply changes
+  <div class="alert alert-success my-4" v-if="saved && restart_supported">
+    <b>Success!</b> Click 'Apply' to restart Sunshine and apply the changes.
+  </div>
+  <div class="alert alert-success my-4" v-if="saved && !restart_supported">
+    <b>Success!</b> Restart Sunshine to apply changes.
+  </div>
+  <div class="alert alert-success my-4" v-if="restarted">
+    <b>Success!</b> Sunshine is restarting to apply changes.
   </div>
   <div class="mb-3 buttons">
     <button class="btn btn-primary" @click="save">Save</button>
+    <button class="btn btn-success" @click="apply" v-if="restart_supported">Apply</button>
```

Comment:
Maybe only show button when settings are saved?

`v-if="saved && restart_supported"`

And maybe when not restarted?

``v-if="saved && restart_supported && !restarted"``